myJoin.pl

```diff
@@ -18,68 +18,67 @@
 my @dup;
 
 #set input flags
-GetOptions ("H" => \$header,
-            "unique" => \$unique,
-            "duplicate" => \$duplicate,
-            "help" => \$help) or die ("Error in command line arguments -h for help\n");
+GetOptions("H" => \$header,
+"unique" => \$unique,
+"duplicate" => \$duplicate,
+"help" => \$help) or die("Error in command line arguments -h for help\n");
 
 #return help flag
-help_me() if ($help);
+help_me()  ($help);
 
-#check for incorrect usage of -d and -u
+#check for incorrect usage -d and -u
```

Comment:
maybe this was intentional. I'll leave it up to you if you want to keep it in a subsequent commit.